.bazelrc

```diff
@@ -3,9 +3,12 @@ try-import submodules/dev-tools/.bazelrc
 # Specific Bazel build/test options.
 build --enable_platform_specific_config
 
-build:windows --compiler="clang-cl"
```

Comment:
Is there a specific reason why you have removed this line?
It looks like it's now compiling with MSVC.

Reply:
Reverted, and tested that the current settings use clang on windows. (Though to be fair I didn't do a similar test removing this line)